src/components/Navbar.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-unused-vars */
+import React, { useContext } from "react";
+import { Link } from "react-router-dom";
+import { CartContext } from "./CartContext";
+import Cart from "./Cart";
+
+function Navbar() {
+  const [cart, setCart] = useContext(CartContext);
```

Comment:
Where do you need `setCart` in this component?

Reply:
removed, thanks